RadixWallet/Clients/EntitiesInvolvedClient/EntitiesInvolvedClient+Interface.swift

```diff
@@ -0,0 +1,34 @@
+// MARK: - EntitiesInvolvedClient
+struct EntitiesInvolvedClient: Sendable {
```

Comment:
This logic was part of `TransactionClient`, extracted it so that it can be reused from `PreAuthorizationClient`.